grammars/jsdoc.cson

```diff
@@ -276,11 +276,11 @@
             \\s*
             (
               # The inner regexes are to stop the match early at */ and to not stop at escaped quotes
-              (?:
+              (?>
```

Comment:
I've tested this locally, and it works. 👍

However, there's no hairy backtracking being performed here, so it's quite strange this is happening...

Reply:
You can play around with the regex debugger at https://regex101.com/r/miqTHE/1 if you want.  The `*` here and the `*` on the everything else line can catastrophically backtrack when there's no `]` or `*/` present.